proprietary/tokens/src/components/ams/logo.tokens.json

```diff
@@ -1,8 +1,9 @@
 {
   "ams": {
     "logo": {
-      "block-size": { "value": "2.5rem" },
+      "block-size": { "value": "{ams.space.grid.md}" },
       "emblem": { "color": { "value": "{ams.color.primary-red}" } },
+      "min-block-size": { "value": "2.5rem" },
```

Comment:
This is a conflict with `block-size` which has a min size of 1rem.

Reply:
The two are intended to cooperate: the minimum height remains ‘40px’ as it was, but the logo is allowed to grow to ‘56px’. The result is 33% top padding, 33% logo, and 33% bottom padding (vertically).